domain/src/main/java/com/example/pinterbest/domain/repositories/AuthRepository.kt

```diff
@@ -4,8 +4,8 @@ import com.example.pinterbest.domain.common.ResponseWithCookie
 import com.example.pinterbest.domain.common.Result
 import com.example.pinterbest.domain.entities.UserLogIn
 import com.example.pinterbest.domain.entities.UserSignUp
-import java.lang.Exception
 import kotlinx.coroutines.flow.Flow
+import java.lang.Exception
```

Comment:
kotlin has its own Exception class, so you can safely delete the line `import java.lang.Exception`

Reply:
Deleted